docs/QUICK_REFERENCE.md

```
@@ -96,7 +96,7 @@ which posts a **verification report comment on the PR**.
   or `verify:create-new-pr` (issue + bootstrapped PR).
 
 **Fleet/aggregate metrics** are surfaced via the weekly summary tracker (issue
-#1796, posted Mondays 06:00 UTC) and the LangSmith dashboard wired by
+#2211, posted Mondays 06:00 UTC) and the LangSmith dashboard wired by
```

markdownlint-cli2 (0.22.1)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)